models/security_groups_test.go

```diff
@@ -376,9 +376,11 @@ var _ = Describe("SecurityGroupRule", func() {
         "destinations": [
           "0.0.0.0-9.255.255.255"
         ],
-        "log": false,
-				"annotations":["quack"]
+		"annotations":["quack"]
       }`
+			// TODO: double-check these fields
+			// protojson doesn't emit this
+			// "log": false,
```

Comment:
Is this TODONE? also see the TODO below

Reply:
Cleaned up in [f7314a9](https://github.com/cloudfoundry/bbs/pull/109/commits/f7314a95de15266d952902af8767dd6b5a741df3)